src/main/webapp/script.js

```diff
@@ -27,16 +27,20 @@ function statLoader() {
   const output = document.getElementById('stat');
   output.innerText = arr[index][0];
 }
+// create score as a global variable
+var score = 0;
 
 // Displays answer after clicking a button
+// Keeps count of the number of correct 
 function buttonPressed(buttonID) {
-  var answer;
```

Comment:
Is it not necessary for this variable to be declared here? Not that familiar with js so just wondering!

Reply:
Oh my bad! I must have accidentally deleted the variable. Yes it is necessary.